### PR TITLE
Add MoonGuard plugin

### PR DESCRIPTION
**Pull request by guetteman**

### CODE REVIEW

**Review by awcodes — APPROVED** [review]

File: content/plugins/moonguard.md
Comment (danharrin) [inline content/plugins/moonguard.md:1]: Please add your author slug to the start of the file names, and the `slug` here, like all the other plugins

**Review by danharrin — CHANGES_REQUESTED** [review]

File: content/plugins/moonguard.md
Comment (guetteman) [reply content/plugins/moonguard.md:1]: Done!

File: content/plugins/luis-guette-moonguard.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: MoonGuard
+slug: luis-guette-moonguard
+author_slug: luis-guette
+categories: ['analytics', 'developer-tool']
+description: Plugin that providers utilities to monitor your app status and performance once it is released to production.
+discord_url: https://discord.gg/vCCy4aJxnY
+docs_url: https://docs.moonguard.dev
```
Comment (awcodes) [inline content/plugins/luis-guette-moonguard.md:8]: Please change this to 'url' instead of 'docs_url' since it is not a markdown file.

**Review by awcodes — CHANGES_REQUESTED** [review]

File: content/plugins/luis-guette-moonguard.md (hunk 1 above)
Comment (guetteman) [reply content/plugins/luis-guette-moonguard.md]: done!

**Review by awcodes — APPROVED** [review]

**Review by saade — APPROVED** [review]

**Comment by saade** [comment]
Hey! Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.

Please let us know when you've addressed these issues.

**Comment by guetteman** [comment]
> Hey! Your plugin's `README` is loaded into the plugin page on the Filament website. There is now a duplicate image on your page, as it also exists in your documentation. You can remove it from the Filament website by giving it a `filament-hidden` class, so then it is only shown once.
> 
> Please let us know when you've addressed these issues.

Done

**Pull request merged**